Added Postman instructions wiki file

f-jovanovic opened the pull request:
Created a translation of the original Postman instructions.
Required images added to the images folder.

Commit:
Added the images for postman wiki page

Commit:
Added Postman instructions md file

bantolov commented on Using-Postman-with-Rhetos.md:
```diff
@@ -0,0 +1,63 @@
+Create a collection for recording all requests. This will simplify and speed up testing of the application.
+First choose the authentication type - NTLM, and enter your domain data (Image 1):
+
+![Postman image 1](https://github.com/f-jovanovic/Rhetos-wiki/blob/master/images/postman-image1.png)
```
Use a relative path of images. See the examples in https://raw.githubusercontent.com/Rhetos/Rhetos-wiki/master/What-is-Rhetos.md

bantolov commented on Using-Postman-with-Rhetos.md:
```diff
@@ -0,0 +1,63 @@
+# Using Postman with the Rhetos framework
+Create a collection for recording all requests. This will simplify and speed up testing of the application.
+First choose the authentication type - NTLM, and enter your domain data (Image 1):
```
The image numbers are not visible in the article, so it is strange to see them referenced by "Image 1/2/3/..".
In this sentence, you could just remove the text "(Image 1)".
In the rest of the article you might change the references to "the image below", "the image above", or something similar.

bantolov commented on Using-Postman-with-Rhetos.md:
```diff
@@ -0,0 +1,63 @@
+![Postman image 1](https://github.com/f-jovanovic/Rhetos-wiki/blob/master/images/postman-image1.png)
+
+Image 2. shows the fields used for testing. Using the left dropdown menu you can select the desired HTTP method (GET, POST, PUT, PATCH, DELETE, etc.), while the center field should contain the server URL (server resource). When the desired method and URL have been entered, hit Send to send the request.
+The difference between PUT and PATCH requests is that **PUT updates the whole entity** while **PATCH only updates the targeted property** of the entity.
```
PATCH is not supported by this REST API, this sentence can be removed.

bantolov commented on Using-Postman-with-Rhetos.md:
```diff
@@ -0,0 +1,63 @@
+![Postman image 3](https://github.com/f-jovanovic/Rhetos-wiki/blob/master/images/postman-image3.png)
+
+The request is sent to Rhetos through a REST service in the following format: `{Rhetos server URL}/Rest/{Module}/{Entity}/`
+Image 3 shows accessing the SustavZaNabavu application through the REST service, and then accessing the module SustavZaNabavu and its entity Adresa. At the the end `/` needs to be added for accessing the resource: `http://localhost/sustavzanabavu/rest/sustavzanabavu/adresa/`
```
The examples should be in English. Please change the examples in the text and in the screenshots, to use the the [Bookstore](https://github.com/Rhetos/bookstore/) demo application.

Some GET and POST examples for the Bookstore application are shown in the chapter "Test and review" in [this article](https://github.com/Rhetos/Rhetos/wiki/Create-your-first-Rhetos-application#test-and-review). Please use **the same URL** here (http://localhost/BookstoreRhetosServer/rest/Bookstore/Book/), since it is also used in other tutorial articles.

bantolov commented on Using-Postman-with-Rhetos.md:
```diff
@@ -0,0 +1,63 @@
+The request is sent to Rhetos through a REST service in the following format: `{Rhetos server URL}/Rest/{Module}/{Entity}/`
+Image 3 shows accessing the SustavZaNabavu application through the REST service, and then accessing the module SustavZaNabavu and its entity Adresa. At the the end `/` needs to be added for accessing the resource: `http://localhost/sustavzanabavu/rest/sustavzanabavu/adresa/`
+
+All the requests are sent in the JSON format. Date properties are formatted in the following way: `\/Date(YYYY-MM-DDTHH:MM:SS.sssZ)\/` (example: `"DatumPredaje":"\/Date(2017-10-31T23:00:00.000Z)\/"`)
```
This is not the correct format for the date properties it Rhetos REST API.
You can just remove this sentence for now. This kind of details can be explained other articles (Rhetos REST API specification).

Commit:
PR changes

Added new images using the Bookstore example, edited the text to reference it

f-jovanovic committed:
Spacing fix

Commit:
Image references edited

Edited the way image files are referenced in the .md to comply with GitHub rules
